package.json

```diff
@@ -119,27 +118,17 @@
     "davidshimjs-qrcodejs": "0.0.2",
     "event-source-polyfill": "0.0.9",
     "highstock-release": "5.0.14",
-    "iscroll": "5.2.0",
     "jquery": "3.2.1",
     "jquery.scrollto": "2.1.2",
     "js-cookie": "2.2.0",
     "kinetic": "5.2.0",
-    "lodash.debounce": "4.0.8",
-    "mobx": "5.0.3",
-    "mobx-react": "^5.4.2",
-    "mobx-utils": "^5.4.0",
     "moment": "2.19.4",
     "promise-polyfill": "6.1.0",
     "prop-types": "15.6.1",
```

Comment:
Can be removed too since it's not used in static?

Reply:
it's used in contract jsx